tools/system_id/README.md

```diff
@@ -0,0 +1,67 @@
+# System ID including PWM, RPM, Thrust, Voltage, Current, and Power
+
+This folder contains scripts to measure propellers and motors using the systemId deck.
+
+## Setup
+
+* Load cells:
+  * 100g: https://www.sparkfun.com/products/14727
+  * 500g: https://www.sparkfun.com/products/14728
+  * 1kg: https://www.adafruit.com/product/4540
+* SystemID board with the following ICs
+  * NAU7802 (thrust)
+  * ACS37800 (power)
+  * QRD1114 (RPM)
+* Calibration weights
+* M2/M3 Nylon screws
+* CF Mount (3D printed)
+
+Mount CF upside down (to avoid downwash)
+
+## Measurement
+
+### Common
+
+1. Mount CF
+2. Switch to crazyflie-lib-python with branch `feature-emergency-stop` in order to be able to CTRL+C a script safely.
```

Comment:
Is this still required?

Reply:
It wasn't required before either but can be handy so I'm going to leave it like that.